docs/guides/testing.md

```diff
@@ -130,7 +135,13 @@ export default config
 
 ### Vitest
 
-In the next steps we are going to setup our Vitest environment in order to mock Zustand
+In the next steps we are going to setup our Vitest environment in order to mock Zustand.
+
+> **Warning:** due to in Vitest you can change the [root](https://vitest.dev/config/#root) you
+> need make sure that you are creating your `__mocks__` folder in the right place. Let's say
+> that you change the **root** to `./src`, that means you need to create `__mocks__` folder
+> under `./src` ending up with `./src/__mocks__` rather than `./__mocks__`. Creating `__mocks__`
+> folder in the wrong place could lead you to some issues while using Vitest.
```

Comment:
Just a few small tweaks here:

> **Warning:** In Vitest you can change the [root](https://vitest.dev/config/#root), because of this, you
> need make sure that you are creating your `__mocks__` directory in the right location. Let's say
> that you change the **root** to `./src`, that means you need to create a `__mocks__` directory
> under `./src`. The end result would be `./src/__mocks__`, rather than `./__mocks__`. Creating a `__mocks__`
> directory in the wrong location could lead to issues while using Vitest.